### PR TITLE
Updated RHODS alerts to point to documentation

### PR DESCRIPTION
**Pull request by lucferbux**
- [X] The commits are squashed in a cohesive manner and have meaningful messages.
- [X] For commits that came from upstream, `[UPSTREAM]` has been prepended to the commit message
- [X] JIRA link(s): https://issues.redhat.com/browse/RHODS-1958
- [x] The Jira story is acked
- [x] An entry has been added to the latest build document in [Build Announcements Folder](https://drive.google.com/drive/folders/1sgkK1WZgGo9CXsLizNe0GbAzVKuSKrZL).
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious)

## Testing steps

- Check that the links are poitint to the correct Runbook.

### CODE REVIEW

File: monitoring/prometheus/prometheus.yaml [hunk 1]
```diff
@@ -592,7 +592,7 @@ data:
         - alert: RHODS Route Error Burn Rate
           annotations:
             message: 'High error budget burn for {{ $labels.route }} (current value: {{ $value }}).'
-            triage: "https://gitlab.cee.redhat.com/service/managed-tenants-sops/-/tree/main/RHODS"
+            triage: "https://gitlab.cee.redhat.com/service/managed-tenants-sops/-/blob/main/RHODS/JupyterHub/rhods-error-burn-rate.md"
             summary: RHODS Route Error Burn Rate
           expr: |
             sum(haproxy_backend_http_responses_total:burnrate6h{route=~"jupyterhub|rhods-dashboard"}) by (route) > (1.00 * (1-0.98000))
```
[the remark this answers is not in the document]
Comment (lucferbux) [reply monitoring/prometheus/prometheus.yaml:598]: great! I think i got it, i need to trigger the alert to test it though

[the remark this answers is not in the document]
Comment (lucferbux) [reply monitoring/prometheus/prometheus.yaml:598]: got it it works

**Comment by anishasthana** [comment]
After upgrade, I can confirm that the yaml is correct.
I tested out the alerts too, the runbook links are good.

**Comment by anishasthana** [comment]
This is only to be merged and included in the 1.3.0 release

**Comment by jgarciao** [comment]
In the case of alerts "User notebook pvc usage at 90%" and "User notebook pvc usage at 100%" the triage url is 
  https://gitlab.cee.redhat.com/service/managed-tenants-sops/-/tree/main/RHODS
but there are no instructions there to increase the PVC

Note: those alerts are not sent to SRE, they are sent to the email address specified when installing RHODS. But it would be nice to have a triage document anyway in the managed-tenants repo. The instructions existed in the old document [JupyterHub Notebook pod has run out of disk space](https://docs.google.com/document/d/1pcYStWmlGN8dtOePyZ5Zy6F07BcHZrraZDYAEx9w_ik/edit?usp=sharing) and also in the users's docs [8.3. I see a database or disk is full error or a no space left on device error when I run my notebook cells](https://access.redhat.com/documentation/en-us/red_hat_openshift_data_science/1/html/getting_started_with_red_hat_openshift_data_science/troubleshooting-common-problems-in-jupyterhub_get-started#i_see_a_emphasis_role_strong_database_or_disk_is_full_emphasis_error_or_a_emphasis_role_strong_no_space_left_on_device_emphasis_error_when_i_run_my_notebook_cells)

**Comment by lucferbux** [comment]
> In the case of alerts "User notebook pvc usage at 90%" and "User notebook pvc usage at 100%" the triage url is https://gitlab.cee.redhat.com/service/managed-tenants-sops/-/tree/main/RHODS but there are no instructions there to increase the PVC
> 
> Note: those alerts are not sent to SRE, they are sent to the email address specified when installing RHODS. But it would be nice to have a triage document anyway in the managed-tenants repo. The instructions existed in the old document [JupyterHub Notebook pod has run out of disk space](https://docs.google.com/document/d/1pcYStWmlGN8dtOePyZ5Zy6F07BcHZrraZDYAEx9w_ik/edit?usp=sharing) and also in the users's docs [8.3. I see a database or disk is full error or a no space left on device error when I run my notebook cells](https://access.redhat.com/documentation/en-us/red_hat_openshift_data_science/1/html/getting_started_with_red_hat_openshift_data_science/troubleshooting-common-problems-in-jupyterhub_get-started#i_see_a_emphasis_role_strong_database_or_disk_is_full_emphasis_error_or_a_emphasis_role_strong_no_space_left_on_device_emphasis_error_when_i_run_my_notebook_cells)

Hi @jgarciao I was about to reply you but you've commented it, this is specific to pagerduty alerts, PVC alerts are mail specific, we brought up that same thing last week.

We can consider opening a new ticket for that specific request (as it's related to email notifications and we should include new instructions) to implement that feature, what do you think @maulikjs @anishasthana ?

**Review by crobby — APPROVED** [review]

**Pull request merged**